app/views/admin/loans/_loans_grid.html.slim

```diff
@@ -32,8 +33,13 @@
       custom_filter: Loan.loan_type_options, detach_with_id: :loan_type_filter do |loan|
       - loan.loan_type
     - g.column name: t('activerecord.attributes.loan.amount'), attribute: 'amount',
-      filter: false do |loan|
-      - format_currency(loan.amount, loan.currency, tooltip: false)
+      filter: false, in_csv: false do |loan|
+      - if loan.amount && loan.currency
```

Comment:
Same as before. If displayed loan amount becomes a method, helper, or some other attribute, it could be used here. However, the specifics are outside the scope of this feature.

I've only fixed this so that CSV export is properly working so that we can see that loan.display_name is exporting properly.